sematic/plugins/publishing/slack.py

```diff
@@ -0,0 +1,114 @@
+# Standard Library
+import logging
+from typing import Type
+
+# Third-party
+import requests
+
+# Sematic
+from sematic.abstract_plugin import (
+    AbstractPlugin,
+    AbstractPluginSettingsVar,
+    PluginVersion,
+)
+from sematic.config.settings import get_plugin_setting
+from sematic.db.models.resolution import Resolution, ResolutionStatus
+from sematic.db.queries import get_run
+from sematic.plugins.abstract_publisher import AbstractPublisher
+from sematic.utils.memoized_property import memoized_property
+
+logger = logging.getLogger(__name__)
+
+_PLUGIN_VERSION = (0, 1, 0)
+_SLACK_HEADERS = {"Content-Type": "application/json"}
+_SLACK_URL_TEMPLATE = "https://hooks.slack.com/services/{webhook_token}"
+# TODO: implement server-side gnostic components that build urls that point to specific
```

Comment:
```suggestion
# TODO: implement server-side agnostic components that build urls that point to specific
```

Reply:
This is a nit, but would you mind using a more common usage synonym like "aware" to avoid confusion? "gnostic" seems to be rarely used outside of religious contexts, whereas "agnostic" sees usage as a term-of-art to mean "unaware."